Formula/rsdns.rb

```diff
@@ -0,0 +1,53 @@
+class Rsdns < Formula
+  desc "Rackspace Cloud DNS CLI"
+  homepage "https://github.com/linickx/rsdns"
+  url "https://github.com/linickx/rsdns/archive/v4.0.tar.gz"
+  sha256 "060bea37a85c9debe59bb23c8a4326e97ced04fe3de15689e34123c978dffb99"
+  head "https://github.com/linickx/rsdns.git"
+
+  bottle :unneeded
+  depends_on "jq"
```

Comment:
Please leave an empty line between `bottle :unneeded` and the first `depends_on`. Thanks!